### DISCUSSION TITLE
Displaying an object with the proper relative size in the given scene

### DISCUSSION THREAD

**Discussion by Domthegrom** [google/model-viewer#2189] [Q&A]
### Description
**QUESTION**

This is more of a question. I've been playin wtih gLTF files and usdz files of objects like a couch or lamp and then rendering them into the scene viewer using the model-viewer tag and every time I try to render in the object it scales up to 100% by default and is extremely large floor to ceiling level large.

I am trying to find a way to set a static object size so that when it renders in it's relative to the real size of the object. Like a couch I wouldn't want to be able to manipulate its size in the scene viewer because I want the size to be true to the real product.

#### Live Demo
I've got a very very simple demo running here: https://admiring-euler-6bba35.netlify.com/

### Browser Affected

Not a bug but I've been testing in: 
- [x ] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] Helios
- [ ] IE
- [x ] Safari

### OS

not a bug but I've been testing in
- [ ] Android
- [ ] iOS
- [ ] Linux
- [x ] MacOS
- [x] Windows

### Versions

- model-viewer: vX.X.X
- three.js: rXX

**Comment by cdata** [comment]
@Domthegrom in general, APIs are starting to align on 1 model unit = 1 meter, and `<model-viewer>` and Scene Viewer align to this as well. So, if you make a model that is 1 unit x 1 unit x 1 unit in your design tool, it will be 1m x 1m x 1m in AR if you are using WebXR or Scene Viewer.

For Apple purposes, I understand that `.usdz` has a way to configure the meters-per-unit in the model, so getting the model to be the right size might come down to how you produce your `.usdz`. We have an outstanding discussion on this topic in #559 where it was requested that we document this detail more thoroughly, but we haven't gotten around to it yet.

**Comment by elalish** [comment]
@Domthegrom In addition to ensuring the model's units are correct, you can also use our `ar-scale` attribute, which when set to `fixed` disables user scaling in SceneViewer so that the model is always at 100%.
